labs/lab00-hello-world/README.md

```diff
@@ -91,7 +91,7 @@ You should have received an invite to log into https://vendor.replicated.com --
 
 You will be in a shared account with all other lab participants -- once you log in, you'll want to select your specific app for the lab:
```

Comment:
Each user that logins in should have their own application view. So this won't be necessary anymore.